Contrib/NP_Score/npscorer.py

```diff
@@ -13,11 +13,13 @@
 # peter ertl, august 2015
 #
 
+from __future__ import print_function
 from rdkit import Chem
 from rdkit.Chem import rdMolDescriptors
 import sys,math,gzip,pickle
+import os.path as op
```

Comment:
I'm not a big fan of this kind of aliasing. It doesn't save you that much typing (particularly here), but it does make the code harder to read.